Sprint-1/exercises/initials.js

```diff
@@ -4,3 +4,7 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string.
 // This should produce the string "CKJ", but you must not write the characters C, K, or J in the code of your solution.
+
+initials = firstName[0] + middleName[0] + lastName[0]
```

Comment:
While declaring your variable 'initials', I didn't see the declaration keyword. Did you miss it?